src/axom/primal/operators/clip.hpp

```diff
@@ -95,6 +95,62 @@ Polygon<T, 3> clip(const Triangle<T, 3>& tri, const BoundingBox<T, 3>& bbox)
   return *currentPoly;
 }
 
+/*!
+ * \brief Clips a 2D subject polygon against a clip polygon in 2D, returning
+ *        the geometric intersection of the subject polygon and the
+ *        clip polygon as a polygon
+ *
+ *  This function clips the subject polygon by the planes obtained from the
+ *  clip polygon's edges (normals point inward). Clipping the
+ *  subject polygon by each plane gives the polygon above that plane.
+ *  Clipping the polygon by a plane involves
+ *  finding new vertices at the intersection of the polygon edges and
+ *  the plane, and removing vertices from the polygon that are below the
+ *  plane.
+ *
+ *
+ * \param [in] subjectPolygon The subject polygon
+ * \param [in] clipPolygon The clip polygon
+ * \param [in] eps The epsilon value
```

Comment:
```suggestion
 * \param [in] eps The epsilon value. Defaults to 1.e-10.
```
You should probably say what the tolerance applies to (could refer to the `clipsPolygonPolygon` methods that uses it if it is documented there).